.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  build_zips:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - otp: 22
+            otp_version: 22.3
+          - otp: 23
+            otp_version: 23.3
+          - otp: 24
+            otp_version: 24.1
+            build_docs: build_docs
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ matrix.otp_version }}
+          version-type: strict
+      - name: Build Elixir Release
+        run: |
+          vsn=$(cat VERSION)
+          make Precompiled.zip
+          mv Precompiled-v${vsn}.zip elixir-otp-${{ matrix.otp }}.zip
+          echo "$PWD/bin" >> $GITHUB_PATH
+      - uses: actions/upload-artifact@v2
+        with:
+          name: elixir-otp-${{ matrix.otp }}.zip
+          path: elixir-otp-${{ matrix.otp }}.zip
```

Comment:
We should upload a zip named `elixir.zip` build against the oldest OTP version.

Reply:
We can rename the `path` but not for the `name` since it's used in the download artifact below.

Reply:
Oh sorry. I misread your comment. Why do we need to use elixir.zip for oldest build?

Reply:
We discussed this and decided not to. Choosing your current OTP version should be preferred whenever possible.